### PR TITLE
Billing Library 5 changes

### PR DESCRIPTION
**Pull request by cb-haripriyan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Flutter SDK
 
+> [!NOTE]  
+> #### Updates for Billing Library 5
+> - SDK Version 1.0: This version uses Google Billing Library 5.2.1 APIs to fetch product information from the Google Play Console and make purchases. If you’re integrating Chargebee’s SDK for the first time, then use this version, and if you’re migrating from the older version of SDK to this version, follow the migration steps in this [document](https://www.chargebee.com/docs/2.0/mobile-playstore-billing-library-5.html).
```
Comment (cb-anurags) [inline README.md:5]: SDK Version 1.0 - Do u wish to add beta here as well? @cb-haripriyan

File: README.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+> - SDK Version 0.4.0: This [version](https://github.com/chargebee/chargebee-flutter/tree/master) includes Billing Library 5.2.1 but still uses Billing Library 4.0 APIs to fetch product information from the Google Play Console and make purchases. This will enable you to list or update your Android app on the store without any warnings from Google and give you enough time to migrate to version 2.0.
```
Comment (cb-anurags) [inline README.md:6]: migrate to version 2.0 - should be version 1.0 chargebee or 5.0 Google ? @cb-haripriyan

**Review by cb-anurags — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (cb-haripriyan) [reply README.md:5]: We can update this.

File: README.md (hunk 2 above)
Comment (cb-haripriyan) [reply README.md]: Will update this.

**Pull request merged**